spec/features/authentication_spec.rb

```diff
@@ -0,0 +1,48 @@
+feature 'authentication' do
```

Comment:
consider possibly splitting this into sign in/ sign out feature files instead of one large one. 

https://github.com/emilyalice2708/chitter-challenge/blob/master/docs/review.md#avoid-rspec-feature-scenarios-organized-like-unit-tests